src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -103,11 +105,22 @@ protected SchemaTypes()
         /// </summary>
         /// <param name="schema">A schema for which this instance is created.</param>
         /// <param name="serviceProvider">A service provider used to resolve graph types.</param>
-#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         public SchemaTypes(ISchema schema, IServiceProvider serviceProvider)
+            : this(schema, serviceProvider, (IEnumerable<IGraphTypeMapping>?)serviceProvider.GetService(typeof(IEnumerable<IGraphTypeMapping>)))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance for the specified schema, and with the specified type resolver.
+        /// </summary>
+        /// <param name="schema">A schema for which this instance is created.</param>
+        /// <param name="serviceProvider">A service provider used to resolve graph types.</param>
+        /// <param name="graphTypeMappings">A service used to map CLR types to graph types.</param>
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+        public SchemaTypes(ISchema schema, IServiceProvider serviceProvider, IEnumerable<IGraphTypeMapping>? graphTypeMappings)
```

Comment:
I don't see usages of this ctor. `Schema` creates `SchemaTypes` explicitly.

Reply:
The standard constructor calls this one and populates the new argument from DI.

Reply:
🤦‍♂️